{ 'arm_test': { 'left_arm': [ { 'duration': 2.024,
                                'name': u'arm_down',
                                'positions': [ -0.2572,
                                               -0.6545,
                                               0.1195,
                                               0.4536,
                                               -0.1432,
                                               -0.0036,
                                               -0.3849],
                                'starttime': 0.0},
                              { 'duration': 1.96,
                                'name': u'arm_up',
                                'positions': [ -0.2543,
                                               0.4185,
                                               -1.9971,
                                               0.4693,
                                               -0.1432,
                                               -0.0031,
                                               -0.3849],
                                'starttime': 3.028}],
                'left_leg': [],
                'neck': [],
                'right_arm': [ { 'duration': 2.024,
                                 'name': u'arm_down',
                                 'positions': [ 0.2572,
                                                0.6545,
                                                0.1195,
                                                -0.4536,
                                                -0.1432,
                                                0.0036,
                                                0.3849],
                                 'starttime': 0.0},
                               { 'duration': 1.96,
                                 'name': u'arm_up',
                                 'positions': [ 0.2543,
                                                -0.4185,
                                                -1.9971,
                                                -0.4693,
                                                -0.1432,
                                                0.0031,
                                                0.3849],
                                 'starttime': 3.028}],
                'right_leg': [],
                'torso': []}}
